cliquet/resource.py

```diff
@@ -18,6 +18,14 @@
     COMPARISON, classname, native_value, decode64, encode64, json,
     current_service, encode_header, decode_header
 )
+from cliquet.events import CollectionChanged
+
+
+# event actions
+ACTION_CREATE = 'create'
```

Comment:
Maybe we can consider using ACTIONS.CREATE as we do for errors here: https://github.com/mozilla-services/cliquet/blob/master/cliquet/errors.py#L8-L28